cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -256,6 +258,15 @@
     # set up user mount namespace
     mount options=(rslave) -> /,
 
+    # set up mount namespace for parallel instances of classic snaps
+    mount options=(rw rbind) /snap/{,*/} -> /snap/{,*/},
+    mount options=(rslave) -> /snap/,
+    mount options=(rw rbind) /var/lib/snapd/snap/{,*/} -> /var/lib/snapd/snap/{,*/},
```

Comment:
We have @SNAP_MOUNT_DIR@, I think this is one of the few reasons why this is a `.in` file and not the real file already. You can use it to reflect the accurate permission for the system.

Reply:
Fixed